### ISSUE TITLE
after I trained 1000 steps, I want to change the learning rate but not start from the 0step.

### ISSUE THREAD

**Issue by RaySteve312** [kijai/ComfyUI-FluxTrainer#40]
after I trained 1000steps, I want to change the learning rate. If I use the default  workflow, I have to restart training from the begining 0 step. Is there a way to change the learning rate and then resume training from 1000 steps?

**Comment by RaySteve312** [comment]
network_weights=path_to_lora , doesn't work with the gradient accumulation steps
![屏幕截图 2024-09-06 185226](https://github.com/user-attachments/assets/731b4265-be0f-4333-83fd-4eba6e632d24)
